README.md

```diff
@@ -74,6 +74,18 @@ Metrics Server | Metrics API group/version | Supported Kubernetes version
 
 [Metrics Server releases]: https://github.com/kubernetes-sigs/metrics-server/releases
 
+### High Availability
+
+Manifests to install an highly available Metrics Server can be found [here](/manifests/high-availability). This configuration can be installed by running:
+
+```shell
+kubectl apply -k ./manifests/high-availability
```

Comment:
We could consider releasing it as a separate components.yaml file with release

Reply:
Good point, I added it to the list of artifacts and changed the installation command